apps/project/schema.py

```diff
@@ -121,6 +126,38 @@ def get_top_entity_contributor(project, Entity):
     ]
 
 
+def get_project_stats_summary(self):
+    projects = Project.get_for_member(self.context.request.user)
+    # Lead stats
+    leads = Lead.objects.filter(project__in=projects)
+    total_leads_tagged_count = leads.annotate(entries_count=models.Count('entry')).filter(entries_count__gt=0).count()
+    total_leads_tagged_and_controlled_count = leads.annotate(
+        controlled_entries_count=models.Count(
+            'entry', filter=models.Q(entry__controlled=True)
+        )
+    ).filter(controlled_entries_count__gt=0).count()
```

Comment:
- Easy read
- Above was missing calculation from previous logic. For future, let's discuss if there are any confusion before implementations.
```suggestion
    total_leads_tagged_count = (
        leads
        .annotate(entries_count=models.Count('entry'))
        .filter(entries_count__gt=0).count()
    )

    # Lead with entries and all entries are controlled(verified)
    total_leads_tagged_and_controlled_count = (
        leads.annotate(
            entries_count=models.Count('entry'),
            controlled_entries_count=models.Count(
                'entry',
                filter=models.Q(entry__controlled=True),
            )
        ).filter(
            entries_count__gt=0,
            entries_count=models.F('controlled_entries_count'),
        ).count()
    )
```